server/app/services/geo/esri/EsriServiceAreaValidationConfig.java

```diff
@@ -0,0 +1,124 @@
+package services.geo.esri;
+
+import com.google.common.collect.ImmutableList;
+import com.typesafe.config.Config;
+import com.typesafe.config.ConfigList;
+import java.util.Optional;
+import javax.inject.Inject;
+
+/** Provides methods for handling Esri address service area validation config. */
+public final class EsriServiceAreaValidationConfig {
+  public Optional<ConfigList> ESRI_ADDRESS_SERVICE_AREA_VALIDATION_LABELS;
+  public Optional<ConfigList> ESRI_ADDRESS_SERVICE_AREA_VALIDATION_VALUES;
+  public Optional<ConfigList> ESRI_ADDRESS_SERVICE_AREA_VALIDATION_URLS;
+  public Optional<ConfigList> ESRI_ADDRESS_SERVICE_AREA_VALIDATION_PATHS;
```

Comment:
**suggestion:** please make these private members

Reply:
updated to private